### ISSUE TITLE
Ansi-tests: The symbol COMMON-LISP::NSUBST has no function bound to it

### ISSUE THREAD

**Issue by kpoeck** [drmeister/clasp#410]
in current dev
````
COMMON-LISP-USER> (let ((tree (list 1 (list 1 2) (list 1 2 3) (list 1 2 3 4))))
	   (nsubst "two" 2 tree))

Condition of type: SIMPLE-PROGRAM-ERROR
The symbol COMMON-LISP::NSUBST has no function bound to it
````

**Comment by kpoeck** [comment]
nsubst-if and nsubst-if-not are implemented in listlib.lsp using nsubst, so they fail too.
`````
COMMON-LISP-USER> (let ((tree (list 1 (list 1 2) (list 1 2 3) (list 1 2 3 4))))
	   (nsubst-if "two" #'(lambda(a)(equal a 2)) tree))

Condition of type: SIMPLE-PROGRAM-ERROR
The symbol COMMON-LISP::NSUBST has no function bound to it
`````
`````
COMMON-LISP-USER> (let ((tree (list 1 (list 1 2) (list 1 2 3) (list 1 2 3 4))))
	   (nsubst-if-not "two" #'(lambda(a)(not (equal a 2))) tree))

Condition of type: SIMPLE-PROGRAM-ERROR
The symbol COMMON-LISP::NSUBST has no function bound to it

Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.
````

**Comment by kpoeck** [comment]
Fixed by PR #428

**Comment by kpoeck** [comment]
Fixed in dev with the merged PR
